profile/README.md

```diff
@@ -6,8 +6,8 @@
 
 Here's how to get started:
 
-- 📚 See available pipelines on the [website](https://nf-co.re/).
-- 👩‍💻 Check out the [documentation](https://nf-co.re/).
-- 🙋‍♀️ Come and say hi on our [Slack](https://nf-co.re/join)!
+- 📚 See available pipelines on the [website](https://nf-co.re/pipelines).
+- 👩‍💻 Check out the [documentation](https://nf-co.re/docs/usage/introduction).
+- 🙋‍♀️ Come and say hi on our [Slack](https://nf-co.re/join/slack)!
```

Comment:
I usually try to avoid going directly to this URL, instead the join page is in the nf-core website and has a button that explains how it works.

```suggestion
- 🙋‍♀️ Come and say hi on our [Slack](https://nf-co.re/join/)!
```